glossary.yml

```diff
@@ -1563,6 +1563,13 @@
       that simulates such a device.
 
 
+- slug: configuration file
+  en:
+    term: "configuration file"
+    def: >
+      A file that specify the parameters and initial settings of software programs. Config files are often used for information subject to changes, such as environment-specific settings.
```

Comment:
```suggestion
      A file that specifies the parameters and initial settings of a software program. Configuration,
      or config, files are often used for information that is subject to change, such as
      environment-specific settings.
```

Reply:
Thank you @baileythegreen. I have made the suggested changes. However, I was uncertain whether, and why, you also suggest I should do a line change after Configuration, ...?

Reply:
The text after my suggestions is only split over 3 lines. `Configuration,`  appears on the first line, but in the suggestion this is wrapped, so might look like it is on a different line. An actual new line will be numbered on the far left, and then have the `+` symbol to indicate a change has been made.

Reply:
It doesn't look like you have committed the changes, actually. You can do so by clicking 'Commit suggestion' below it.
<img width="719" alt="Screen Shot 2021-02-03 at 11 41 45" src="https://user-images.githubusercontent.com/12277715/106742357-d7c01180-6614-11eb-84c5-3f05eee480e4.png">

Reply:
Thanks for clarifying. I did not commit the changes yet, I wanted to wait till I had heard from you. Thanks for pointing out the "commit suggestion" option, it seems there is a time limit for using that? It does not matter, it is fine that I get to practice this a few times.

Reply:
I don't think there is a time limit. That would be weird. Is that option not available to you?

As far as a new PR, no. Once a PR is submitted, you can still make commits on top of it by pushing to the same branch: in this case, add_config_file. This is a sensible thing to do, so long as the new commits are related to the original intent of the PR. (In this case they are.)

Reply:
I think perhaps the problem was that I had already committed the changes (commit d56e9fd9dc7aaa428f94083e4ee437bb8e8011e9) before I noticed the "commit suggestion" button. When I hover the mouse there it says "outdated suggestions cannot be applied". 

I really hope everything is in order now. Thank you for your patience with me.

Reply:
Oh, I see; yes, that is indeed what happened. For some reason it wasn't clear on my end yesterday. Anyway, in future, if here are specific suggestions, as in this case, you can use that button to commit them, which might be easier.

But if there are larger changes, or just vague comments about changes requested, then the way you did it might be necessary/preferable.